chef_master/source/index.rst

```diff
@@ -301,7 +301,7 @@ Chef DK
 
 `knife spork </plugin_knife_spork.html>`__
 
-**knife supermarket**: `supermarket download </plugin_knife_supermarket.html#download>`__ | `supermarket install </plugin_knife_supermarket.html#install>`__ | `supermarket list </plugin_knife_supermarket.html#list>`__ | `supermarket search </plugin_knife_supermarket.html#search>`__ | `supermarket share </plugin_knife_supermarket.html#share>`__ | `supermarket show </plugin_knife_supermarket.html#show>`__ | `supermarket unshare </plugin_knife_supermarket.html#unshare>`__
+**knife supermarket**: `supermarket download </knife_supermarket.html#download>`__ | `supermarket install </knife_supermarket.html#install>`__ | `supermarket list </plugin_knife_supermarket.html#list>`__ | `supermarket search </plugin_knife_supermarket.html#search>`__ | `supermarket share </plugin_knife_supermarket.html#share>`__ | `supermarket show </plugin_knife_supermarket.html#show>`__ | `supermarket unshare </plugin_knife_supermarket.html#unshare>`__
```

Comment:
Doesn't the rest of the links listed here need to be changed to "knife_supermarket" for every time I see "plugin_knife_supermarket"? I only see changes to 2 out of 7 links listed here.

Reply:
yes these should be updated as well